content/pt/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,156 @@
+```
+
+Logs não são suficientes para rastrear a execução do código, pois geralmente não
+apresentam informações contextuais, como de onde foram chamados.
+
+Eles se tornam muito mais úteis quando incluídos como parte de um
+[trecho](#spans) ou quando são correlacionados com um rastro e um trecho.
+
+Para mais informações sobre logs e como eles se relacionam com o OpenTelemetry,
+consulte [Logs](/docs/concepts/signals/logs/).
+
+### Trechos
+
+Um **trecho** representa uma unidade de trabalho ou operação. Trechos rastreiam
+operações específicas que uma requisição realiza, mostrando o que aconteceu
+durante o tempo em que essa operação foi executada.
+
+Um trecho contém nome, dados relacionados ao tempo,
+[mensagens de log estruturadas](/docs/concepts/signals/traces/#span-events) e
+[outros metadados (ou seja, Atributos)](/docs/concepts/signals/traces/#attributes)
+para fornecer informações sobre a operação sendo rastreada.
+
+#### Atributos de trecho
+
+Os atributos de trechos são metadados anexados a um trecho.
+
+A tabela a seguir contém exemplos de atributos de trecho:
+
+| Chave                       | Valor                                                                              |
+| :-------------------------- | :--------------------------------------------------------------------------------- |
+| `http.request.method`       | `"GET"`                                                                            |
+| `network.protocol.version`  | `"1.1"`                                                                            |
+| `url.path`                  | `"/webshop/articles/4"`                                                            |
+| `url.query`                 | `"?s=1"`                                                                           |
+| `server.address`            | `"example.com"`                                                                    |
+| `server.port`               | `8080`                                                                             |
+| `url.scheme`                | `"https"`                                                                          |
+| `http.route`                | `"/webshop/articles/:article_id"`                                                  |
+| `http.response.status_code` | `200`                                                                              |
+| `client.address`            | `"192.0.2.4"`                                                                      |
+| `client.socket.address`     | `"192.0.2.5"` (o cliente passa por um proxy)                                       |
+| `user_agent.original`       | `"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:72.0) Gecko/20100101 Firefox/72.0"` |
+
+Para mais informações sobre trechos e como eles se relacionam com o
+OpenTelemetry, consulte [Trecho](/docs/concepts/signals/traces/#spans).
+
+### Rastros distribuídos {#the-untranslated-english-title}
```

Comment:
```suggestion
### Rastros distribuídos
```

Reply:
```suggestion
### Rastros distribuídos {#distributed-traces}
```

as of today we keep the anchors untranslated.